UnitTest/tests/test_developid_1-definition.txt

```diff
@@ -228,7 +228,7 @@ defmod test_developid_1 UnitTest dummyDuino (
 	$subTestName='SIGNALduino_IdList developId "m" dispatch enabled/disabled tests';
 	subtest $subTestName => sub {
 		Log3 $name, 3 ,$subTestName." |->";
-		$logfilter="IdList, development protocol is active";
+		$logfilter="IdList, development Protocol is active";
```

Comment:
protocol schreibt man auf Englisch klein oder?

Reply:
Man schreibt alles in Englisch klein was nicht am Anfang steht ;) lernte ich mal. Da leider es auch da Differenzen bzw. Meinungen von „nicht English sprechenden“ gibt, so passte ich es der großen Form von „decoded Protocol ... „ an.

Hauptgrund ist Einheitlichkeit. Gern mache ich alles auch klein.

Reply:
Ich hab das auch so gelernt, dass nur "Namen" und das Wort am Anfang eines Satzes groß geschrieben wird. Ist mir halt jetzt hier gerade aufgefallen, sicher gibt es da noch viel mehr Stellen die nicht richtig sind ;)

Reply:
Dann stelle ich die geänderten Stellen auf klein um.
Die anderen Positionen schaue ich im nächsten Step durch.

Als ToDo für v3.9 können wir ja eine Translate Variante implementieren ;)